python/async_pipeline.py

```diff
@@ -0,0 +1,150 @@
+"""

```

Comment:
what's the purpose of this file ?

Reply:
its 

from async_pipeline import AsyncPipeline  in ,notebook_utils fil ewe imported this function

without this im getting this error

\notebook_utils.py", line 23, in <module>
    from async_pipeline import AsyncPipeline
ModuleNotFoundError: No module named 'async_pipeli